Dev/VideoChatClient/Output/AudioOutput.cpp

```diff
@@ -0,0 +1,13 @@
+#include "AudioOutput.hpp"
+
+
+void AudioOutput::show_output(Buffer& out)
+{
+	sf::SoundBuffer buffer{};
+	buffer.loadFromMemory(out.data(), out.size());
+
+	sf::Sound sound{};
+	sound.setBuffer(buffer);
+	sound.play();
+	Sleep(buffer.getDuration().asMilliseconds());
```

Comment:
I believe you that how it works, explain me later why sleeping, I infer the play doesn’t block…